scripts/db/src/get-user.ts

```diff
@@ -26,44 +26,68 @@ async function getByEmail(email: string, callback: DbScriptCallback) {
   //     callback(new Error("my error message"));
   try {
     /** Get required dependencies */
-    const { MongoClient } = require('mongodb@4.1.0')
+    const { Client: PGClient } = require('pg@8.7.1')
 
-    const { MONGO_URI, MONGO_DB_NAME } = configuration
+    const {
+      POSTGRES_USERNAME,
+      POSTGRES_PASSWORD,
+      POSTGRES_HOST,
+      POSTGRES_DATABASE,
+      POSTGRES_PORT,
```

Comment:
These variables used to be used to connect to parfit, repurposing them to connect to the forum dev, staging, prod databases